src/plugins/StyledComponentsPlugin.ts

```diff
@@ -0,0 +1,28 @@
+import { WorkFlowContext } from "./../core/WorkflowContext";
+import { Plugin } from "../core/WorkflowContext";
+
+const createStyledComponentsTransformer = require('typescript-plugin-styled-components').default;
```

Comment:
Please move that into `init` or some place else inside the plugin, otherwise it will yield an immediate error